src/Icons/doc/index.rst

```diff
@@ -19,10 +19,60 @@ Installation
 
     $ composer require symfony/ux-icons
 
-Icons?
+Icons
 ------
 
-No icons are provided by this package but there are several ways to include and render icons.
+The ``symfony/ux-icons`` package aspires to offer simple and easy-to-use ways to
+handle SVG icons in your Symfony application.
+
+SVG Icons
+~~~~~~~~~
+
+[SVG](https://en.wikipedia.org/wiki/SVG) (Scalable Vector Graphics) is an XML-based
+vector image format. SVGs are a great way to add scalable, resolution-independent
+graphics to your website. They can be styled with CSS, combined, animated, reused..
+for a fraction of the file size of bitmap images.
+
+Making them the perfect choice for icons.
+
+
+Icon Sets
```

Comment:
I feel that in this section there are too many h3 and h4 subsections. Maybe we can simplify things a bit?

Reply:
Done, i'll continue tomorrow i'm KO !